docs/userguide/constraints.md

```diff
@@ -222,10 +233,123 @@ DiscreteLinkedParametersConstraint(
 | 3 | Octanol           | Octanol             |                                         |
 
 ### ``DiscreteDependenciesConstraint``
-Content coming soon...
+A dependency is a situation where parameters depend on other parameters.
+Let's say an experimental setup has a parameter called "Switch" which turns on 
+pieces of equipment that are optional.
+This means the other parameters (called ``affected_parameters``) are only relevant if 
+the "Switch" parameter has the value "on". If the switch is "off", the affected parameters 
+are irrelevant.
+
+You can specify such a dependency with the ``DiscreteDependenciesConstraint``, which 
+requires:
+1) A list ``parameters`` with the names of the parameters upon which others depend.
+2) A list of ``Condition``'s which specify the values of the corresponding entries in 
+   ``parameters`` which "activate" the dependent parameters.
+3) A list of lists, each containing the ``affected_parameters``, which become relevant 
+   only if the corresponding entry in ``parameters`` is active as specified by the entry in ``conditions``.
+
+Internally, BayBE drops entries from the ``SearchSpace`` where affected parameters are 
```

Comment:
Is information like that relevant for the user guide? Not sure about that either, but that feels like information too specific for a basic user guide.

Reply:
I guess after our general discussion this can be resolved as this ind of info has a place in our planned user guides?